docs/toc.md

```diff
@@ -164,6 +164,14 @@
 ## [MarkdownParser](parsers/MarkdownParser.md)
 ## [RSSParser](parsers/RSSParser.md)
 
+# High performance
+## [Introduction](high-performance/Introduction.md)
+## Buffers
+### [MemoryOwner&lt;T>](high-performance/MemoryOwner.md)

```

Comment:
Don't think you convert here and use the `<` directly:

![image](https://user-images.githubusercontent.com/24302614/82507080-2ee97a80-9ab6-11ea-99f4-1a71f2b7a1f7.png)

Reply:
Ah, you're right - I was doing this because using `<T>` wasn't working in the markdown preview, but I forgot this file isn't actually displayed as markdown at all in the docs 😄

Fixed in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/301/commits/4d21229463c7fa67675c898e4cac9cf07c9bfa18.